kerastuner/tuners/finetuner.py

```diff
@@ -0,0 +1,295 @@
+from tensorflow import keras
+from tensorflow.keras import layers
+from ..engine import hyperparameters as hp_module
+from ..engine import multi_execution_tuner
+from ..engine import oracle as oracle_module
+from ..engine import trial as trial_lib
+
+class FineTuner(multi_execution_tuner.MultiExecutionTuner):
```

Comment:
For this PR, let's wait on the outcome of the design doc on how to best handle transfer learning